state.php

```diff
@@ -28,9 +39,7 @@
 
     header("HTTP/1.1 200 OK");
     die();
-
-} else if ($_SERVER['REQUEST_METHOD'] == 'GET') {
-
+} elseif ($_SERVER['REQUEST_METHOD'] == 'GET') {
```

Comment:
else <should be space> if

Reply:
By the PSR2 standard both should be elseif. Together.